numba/cuda/tests/cudadrv/test_cuda_ndarray.py

```diff
@@ -142,6 +142,28 @@ def test_devicearray_transpose_wrongaxis(self):
                 'axis 2 is out of bounds for array of dimension 2',  # sim
             ])
 
+    def test_devicearray_view_ok(self):
+        original = np.array(np.arange(12), dtype="i2").reshape(3, 4)
+        array = cuda.to_device(original)
+        self.assertTrue(np.all(
+            array.view("i4").copy_to_host() == original.view("i4")
+        ))
+        self.assertTrue(np.all(
+            array.view("u4").copy_to_host() == original.view("i4")
+        ))
+        self.assertTrue(np.all(
+            array.view("i8").copy_to_host() == original.view("i8")
+        ))
+        self.assertTrue(np.all(
+            array.view("f8").copy_to_host() == original.view("f8")
+        ))
+
+    def test_devicearray_view_err(self):
```

Comment:
It may be worth checking the exception message in this test as well (e.g. as in `test_devicearray_transpose_wrongdim` above)

Reply:
Ok will add.

Reply:
Done. Thanks!